pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSetMetaData.java

```diff
@@ -37,25 +37,22 @@ public PgResultSetMetaData(BaseConnection connection, Field[] fields) {
     fieldInfoFetched = false;
   }
 
-  /*
-   * Whats the number of columns in the ResultSet?
+  /**
+   * What's the number of columns in the ResultSet?
```

Comment:
Is the javadoc intentionally different from the base class documentation?

I think it might be better to mark the method with `@Override` and trim the documentation provided it adds nothing special to the `ResultSetMetaData` documentation.

AFAIK, pgjdbc build procedure inherits JDK documentation.

Reply:
AFAICT the only reason to have documentation in the implementation is for clarity when reading the implementation. javadoc should automatically inherit the interface docs. Unless of course as you mentioned there is something special about the implementation.